rules/windows/execution_command_prompt_connecting_to_the_internet.toml

```diff
@@ -23,11 +48,65 @@ index = ["winlogbeat-*", "logs-endpoint.events.*", "logs-windows.*"]
 language = "eql"
 license = "Elastic License v2"
 name = "Command Prompt Network Connection"
+note = """## Triage and analysis
+
+### Investigating Command Prompt Network Connection
+
+Attackers commonly transfer tooling or malware from external systems into a compromised environment using the command and control channel. However, they can also abuse signed utilities to drop these files.
+
+This rule looks for a network connection to an external address from the `cmd.exe` utility followed by a file creation, which can indicate the abuse of the utility to download malicious files and tools.
```

Comment:
Looking at the EQL, I don't think this rule looks for a file creation after the network connection event

Reply:
This may also impact the file analysis portion of the investigation guide instructions, as this will likely trigger on many occasions. If the file creation event is added to the sequence rule, the IG is good

Reply:
Yeah, this was based on the assumption of a tuning I was doing at the time. Nice catch!
I'll adjust it